src/pymovements/utils/__init__.py

```diff
@@ -17,6 +17,7 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+# pylint: disable=cyclic-import
```

Comment:
unfortunately I did not find a way to resolve this cyclic import in this PR.

it seems to be related to `stimulus.text` importing `utils.plotting`, and `utils.aois` importing from `stimulus.text`

it is completely resolved in the follow-up PR #1081